harness/determined/exec/prep_container.py

```diff
@@ -255,6 +267,9 @@ def do_proxy(sess: Session, allocation_id: str) -> None:
 
     if args.resources:
         det.ResourcesInfo._by_inspection()._to_file()
+        from determined.gpu import check_for_gpu_processes
```

Comment:
always prefer importing at the very top, without very good reason to do otherwise.

In this case, `determined.gpu` is a pretty trivial file to import, so there's not a good reason.

Also, for python imports, please stick to the [google style guide](https://google.github.io/styleguide/pyguide.html#22-imports).  So this should be:

```python
from determined import gpu
...
gpu.check_for_gpu_processes
```